docs/content/how-to/automated-machine-management/performance-profiling.md

```diff
@@ -0,0 +1,83 @@
+# Manage node-level performance profiling with Performance Profile Controller
+
+## Intro
+
+The `Performance Profile controller`, formerly known as `Performance-Addon Operator`, is now part of `Node Tuning Operator`.
+[Performance Profile controller](https://github.com/openshift/cluster-node-tuning-operator/blob/master/docs/performanceprofile/performance_controller.md) allows you to optimize nodes in your hosted cluster for applications sensitive to CPU and network latency.
+
+In HyperShift, node profiling can be configured by creating ConfigMaps which contain [`PerformanceProfile`](https://docs.openshift.com/container-platform/4.11/rest_api/node_apis/performanceprofile-performance-openshift-io-v2.html) objects, and referencing these ConfigMaps in your NodePools.
+
+## Steps
+
+1. Create a ConfigMap which contains a valid `PerformanceProfile` manifest and reference it in a NodePool.
+   The example `PerformanceProfile` manifest defines several parameters like how to partition your CPUs into housekeeping and workload partitions, what topology policy to use, and more.
+   Save this ConfigMap as `perfprof-1.yaml`.
+
+    ```yaml
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: perfprof-1
+      namespace: clusters
+    data:
+      tuned: |
+        apiVersion: performance.openshift.io/v2
+        kind: PerformanceProfile
+        metadata:
+        name: performance
+        spec:
+            additionalKernelArgs:
+            - nmi_watchdog=0
+            - audit=0
+            - mce=off
+            - processor.max_cstate=1
+            - intel_idle.max_cstate=0
+            - idle=poll
+            cpu:
+                isolated: "1,3,5,7,9,11,13,15,17,19-39,41,43,45,47,49,51,53,55,57"
+                reserved: "0,2,4,6,8,10,12,14,16,18,40,42,44,46,48,50,52,54,56,58"
+                offlined: "59-79"
+            nodeSelector:
+                node-role.kubernetes.io/worker-cnf: ""
+            numa:
+                topologyPolicy: single-numa-node
+            realTimeKernel:
+                enabled: true
+            workloadHints:
+                highPowerConsumption: true
+                realtime: true
+    ```
+
+    > **_NOTE:_** See [PerformanceProfile Creator](https://docs.openshift.com/container-platform/4.11/scalability_and_performance/cnf-create-performance-profiles.html) for help in creating this `PerformanceProfile` manifests.
```

Comment:
Should this link be updated to a newer version of OCP? Currently set at 4.11.

Reply:
Sure.
Changed it to `latest` instead of using a explicit version